### PR TITLE
introduces REST based view and develops corresponding controllers

### PR DESCRIPTION
**Pull request by mateuszlichon**

### CODE REVIEW

File: src/main/java/com/pollapp/controller/PollController.java [hunk 1]
```diff
@@ -3,24 +3,34 @@
...
+    public List<Poll> getClosedPoll() {
+        return pollRepository.findAll();
     }
 
-    @PostMapping("")
-    public Poll createPoll() {
-        // TODO
-        return null;
+    @PostMapping("/create")
+    public void createPoll(@RequestParam Poll poll) {
```
Comment (przemyslawkonik) [inline src/main/java/com/pollapp/controller/PollController.java]: @PostMapping("/") is enough to keep api clean

**Review by przemyslawkonik — COMMENTED** [review]
Well done. One objection to api.

File: src/main/java/com/pollapp/controller/PollController.java (hunk 1 above)
Comment (mateuszlichon) [reply src/main/java/com/pollapp/controller/PollController.java]: Thanks. I've changed it.

Comment (przemyslawkonik) [reply src/main/java/com/pollapp/controller/PollController.java]: :+1: 
 I also updated url in script.

**Pull request merged**